docs/components/ActionLabel/ActionLabel.stories.mdx

```diff
@@ -1,16 +1,16 @@
-import { Canvas, Meta } from "@storybook/addon-docs";
+import { Meta } from "@storybook/addon-docs";
 import { ActionLabel } from "@jobber/components-native";
 
 <Meta title="Components/Actions/ActionLabel/Docs" component={ActionLabel} />
 
 # Action Label
 
-The `ActionLabel` is a base component that is widely used in areas have an
+The `ActionLabel` is a base component that is widely used in areas that have an
```

Comment:
What do we think about keeping only props within backticks? 
This is different for each component page and we should be streamlined on this one way or another.

Reply:
```suggestion
The ActionLabel is a base component that is widely used in areas that have an
```